Mobile Buy SDK/Mobile Buy SDK/Payment Providers/BUYApplePayPaymentProvider.m

```diff
@@ -242,7 +242,7 @@ - (void)paymentAuthorizationViewController:(PKPaymentAuthorizationViewController
 			default:
 				break;
 		}
-		
+		self.checkout = self.applePayAuthorizationDelegate.checkout;
```

Comment:
Your right, ideally it shouldnt be public.  There is no deterministic way that the PaymentProvider would know that the checkout has changed on the Delegate.  If we embrace that the checkout property is public readonly because we know it can and will be updated, KVO may be a good solution.